internal/dataplane/sendconfig/sendconfig.go

```diff
@@ -91,23 +121,37 @@ func onUpdateInMemoryMode(
 	log logrus.FieldLogger,
 	state *file.Content,
 	client InMemoryClient,
-) error {
+) (error, []ResourceError, error) {
```

Comment:
Could we add comments to interpret what are the return values for? two `error` return values are a bit confusing.

Reply:
The added docstring is a beast of a sentence, but hopefully helps :smile: